app/mailers/application_mailer.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationMailer < ActionMailer::Base
   domain = ENV.fetch('MAILGUN_DOMAIN', 'roqua.nl')
-  default from: "noreply@#{domain}"
+  default from: "CATja Screening <noreply@#{domain}>"
```

Comment:
Bij de bijlage had ik de `-` gezet om geen spaties in bestandsnamen te krijgen, vandaar.
Laten we de opdrachtgever vragen hoe ze dit willen hebben. :)